packages/utils-query-operation/lib/Utils.ts

```diff
@@ -110,3 +110,20 @@ export function removeOperationSource(operation: Algebra.Operation): void {
     delete operation.metadata;
   }
 }
+
+/**
+ * Sets KEY_CONTEXT_WRAPPED_QUERY_OPERATION to the operation being executed.
+ * @param operation The query operation.
+ * @param context The current action context.
+ * @returns A new action context with the operation marked as wrapped.
+ */
+export function setContextWrapped(operation: Algebra.Operation, context: IActionContext): IActionContext {
```

Comment:
Just a question, as you probably have a better view on this than me:
Is this util function (and context key) required here?
Can we not move it to the wrapper actor itself?
Because I suspect that only that wrapper actor will ever need to call this and use this context key?

The same applies to the util function in `ActorRdfJoin`.

Reply:
Yep, I agree.